src/main/java/uk/gov/companieshouse/extensions/api/common/ExtensionDateUtils.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.extensions.api.common;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+
+public class ExtensionDateUtils {
```

Comment:
Just a general point really - perhaps something to consider for the future/next project - but rather than use static methods for things like this we could make use of Java functional interfaces and then auto-wire class instances where the functionality is needed. Would result in a class per function which could then easily be used as a lambda. e.g. in a stream, if needed.